index.html

```diff
@@ -159,15 +159,18 @@ <h4 class="">Skills</h4>
         </section>
         
         
-    </footer>
-        <h2 class="head-contact">
-        <form action="" class="form-contact">
-            <input type="text" placeholder="  Full name" name="name" id="name">
-            <input type="email" placeholder="  Email address" name="email" id="email">
-            <textarea name="message" id="message" cols="30" rows="10">  Enter text here</textarea>
+    <footer class="ContacMe">
+        <p class="head-contact">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/xjvznrod" method="POST" class="form-contact">
+            <input type="text" placeholder="  Full name" name="name" id="name" required maxlength="30">
+            <input type="email" placeholder="  Email address" name="email" id="email" required>
+            <textarea name="message" id="message" cols="30" rows="10" required maxlength="500">  Enter text here</textarea>
+            <button  class="btn-big" id="resume">Get In touch</button>
```

Comment:
- [ ] [OPTIONAL] If you implement `button` element with type `submit` it would be best practice 👍🏼